### PR TITLE
Added to Learning Objectives in 03 and small rewording in 04

### PR DESCRIPTION
**Pull request by SeanRGB**
Small change. I hope this is okay. Still getting the hang of how things work here at SWC.

### CODE REVIEW

File: 03-create.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ minutes: 10
 ---
 > ## Learning Objectives {.objectives}
 > 
-> *   Create a local Git repository.
+> *   Learn how to create a local Git repository using `git init`.
```
Comment (iglpdc) [inline 03-create.md]: I prefer the original version: we are writing the learning objectives as "In this topic, you will learn to ..."

File: 03-create.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ minutes: 10
...
+> *   Learn how to locate `.git` directory and understand what its purpose is.
```
Comment (iglpdc) [inline 03-create.md]: This should be added, but could you write it as "Understand the purpose of the `.git` directory"

**Comment by iglpdc** [comment]
Thanks @SeanRGB ! Could you make the small change I suggested in the first commit?

**Comment by rgaiacs** [comment]
Thanks @SeanRGB ! Could you make the small change I suggested in the first commit?

**Comment by kekoziar** [comment]
This PR is stale and several years out of date.  Please update your branch with the main lesson.

In terms of learning objectives, they should be measureable and stated in an active voice. You can't measure "to learn" or "understand," so the learning objectives should be "create a local Git repository", "locate the `.git` directory," and "describe the purpose of the `.git` directory"

**Comment by kekoziar** [comment]
PR is stale and superseded by subsequent PRs.  Closing.

**Pull request closed without merging**